src/sentry/templatetags/sentry_assets.py

```diff
@@ -35,3 +35,8 @@ def crossorigin():
         # They share the same domain prefix, so we don't need CORS
         return ''
     return ' crossorigin="anonymous"'
+
+
+@register.simple_tag
+def locale_js_url(request):
+    return get_asset_url("sentry", "dist/moment/locale/" + request.LANGUAGE_CODE + ".js")
```

Comment:
maybe just assert language code isin settings.LANGUAGES (or whatever that list is)